sklearn/mixture/dpgmm.py

```diff
@@ -501,10 +501,10 @@ def fit(self, X):
             List of n_features-dimensional data points.  Each row
             corresponds to a single data point.
         """
-        self.random_state = check_random_state(self.random_state)
+        self.random_state_ = check_random_state(self.random_state)
```

Comment:
We used this pattern in other places, I don't remember where, though. The previous pattern of just overwriting the parameter is clearly not what we want, though.